Sources/PackageDescription/Resource.swift

```diff
@@ -35,25 +50,38 @@ public struct Resource: Encodable {
         self.localization = localization
     }
 
-    /// Apply the platform-specific rule to the given path.
```

Comment:
I think using "a" sounds to me as if I could pick a rule from a list of rules. Maybe "Applies platform specific rules" would be better?